app/config.default.coffee

```diff
@@ -1,16 +1,37 @@
-SERVER_ROOT = process.env.SERVER_ROOT or "http://beta.digitransit.fi"
+find = require 'lodash/collection/find'
+
+CONFIG = process.env.CONFIG or 'default'
+API_URL = process.env.API_URL or 'http://dev.digitransit.fi'
+APP_PATH = process.env.APP_CONTEXT or ''
+PIWIK_ADDRESS = process.env.PIWIK_ADDRESS or ''
+PIWIK_ID = process.env.PIWIK_ID or ''
+SENTRY_DSN = process.env.SENTRY_DSN or ''
+NODE_ENV = process.env.NODE_ENV or 'development'
+PORT = process.env.PORT or 8080
+
+appRoot = process.cwd() + '/'
+if process.env.NODE_ENV != 'development'
+  stats = require('../stats.json')
+  svgSprite = APP_PATH + '/' + find(stats.modules, 'id', "./static/svg-sprite.svg").assets[0] #TODO: Cache this
+
 module.exports =
+  PIWIK_ADDRESS: "#{PIWIK_ADDRESS}"
+  PIWIK_ID: "#{PIWIK_ID}"
+  SENTRY_DSN: "#{SENTRY_DSN}"
+  NODE_ENV: "#{NODE_ENV}"
+  PORT: PORT
+  CONFIG: "#{CONFIG}"
   URL:
-    SERVER_ROOT: SERVER_ROOT
-    OTP: "#{SERVER_ROOT}/otp/routers/finland/"
-    MAP: "#{SERVER_ROOT}/hsl-map/"
+    API_URL: "#{API_URL}"
+    OTP: "#{API_URL}/otp/routers/finland/"
+    MAP: "#{API_URL}/hsl-map/"
     MQTT: "ws://213.138.147.225:1883"
-    ALERTS: "#{SERVER_ROOT}/hsl-alert/"
+    ALERTS: "#{API_URL}/hsl-alert/"
     FONT: "http://fonts.googleapis.com/css?family=Lato:300,400,900%7CPT+Sans+Narrow:400,700"
-    REALTIME: "#{SERVER_ROOT}/navigator-server"
-    PELIAS: "#{SERVER_ROOT}/pelias/v1/autocomplete"
-    PELIAS_REVERSE_GEOCODER: "#{SERVER_ROOT}/pelias/v1/reverse"
-  ROOT_PATH: process.env.ROOT_PATH or ''
```

Comment:
There are still references to ROOT_PATH:
- https://github.com/HSLdevcom/digitransit-ui/blob/split-vendor-chunks/app/client.cjsx#L55
- https://github.com/HSLdevcom/digitransit-ui/blob/split-vendor-chunks/app/server.cjsx#L112
- https://github.com/HSLdevcom/digitransit-ui/blob/split-vendor-chunks/app/server.cjsx#L125
- https://github.com/HSLdevcom/digitransit-ui/blob/split-vendor-chunks/webpack.config.js#L113

Should these be:
a) removed?
b) renamed to API_ROOT?
c) something else?